en/micro-integrator/docs/use-cases/examples/message_transformation_examples/json-to-soap-conversion.md

```diff
@@ -0,0 +1,195 @@
+# Converting JSON to SOAP 
+
+Let's consider a scenario where you have a SOAP-based backend (that can only recieve SOAP request) and a JSON client. When the JSON client sends a message to the SOAP client, the proxy service in the Micro Integrator should convert the JSON message to SOAP.
+
+The following examples explain different methods of converting JSON messages to SOAP using the Micro Integrator.
+
+## Using the PayloadFactory Mediator
+
+Let's convert JSON messages to SOAP using the [PayloadFactory mediator](../../../../references/mediators/payloadFactory-Mediator).
+
+### Synapse configuration
+Following is a sample proxy service configuration that we can used to implement this scenario. See the instructions on how to [build and run](#build-and-run) this example.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy xmlns="http://ws.apache.org/ns/synapse"
+       name="JSONToSOAP"
+       startOnLoad="true"
+       statistics="disable"
+       trace="disable"
+       transports="http,https">
+   <target>
+      <inSequence>
+         <payloadFactory media-type="xml">
+            <format>
+               <soapenv:Envelope xmlns:soapenv="http://www.w3.org/2003/05/soap-envelope"
+                                 xmlns:ns="http://www.viewstar.com/webservices/2002/11">
+                  <soapenv:Header/>
+                  <soapenv:Body>
+                     <ns:placeOrder>
+                        <ns:order>
+                            <ns:symbol>$1</ns:symbol>
+                            <ns:price>$2</ns:price>
+                            <ns:quantity>$3</ns:quantity>
+                        </ns:order>
+                    </ns:placeOrder>
+                  </soapenv:Body>
+               </soapenv:Envelope>
+            </format>
+            <args>
+               <arg evaluator="json" expression="$.placeOrder.order.symbol"/>
+               <arg evaluator="json" expression="$.placeOrder.order.price"/>
+               <arg evaluator="json" expression="$.placeOrder.order.quantity"/>
+            </args>
+         </payloadFactory>
+         <log level="full"/>
+         <send>
+              <endpoint>
+                  <address uri="http://localhost:9000/services/SimpleStockQuoteService"/>
+              </endpoint>
+          </send>
```

Comment:
The soapUI/Postman client will hang if we don't reply back

Reply:
resolved